src/components/FormCK/FormCK.js

```diff
@@ -0,0 +1,196 @@
+import React, { useState } from 'react';
+import { useStaticQuery, graphql } from 'gatsby';
+import { Box } from 'rebass';
+import { useForm } from "react-hook-form";
+import { Input } from '@rebass/forms';
+import fetch from 'node-fetch';
+import styled from "@emotion/styled";
+import DefaultBeforeCopy from './DefaultBeforeCopy';
+import DefaultLeftCopy from './DefaultLeftCopy';
+import ThankYou from './ThankYou';
+
+const FormCK = ({ copyBefore, submitText, formId, children}) => {
```

Comment:
So to make a custom form I would:

1. create beforeCopy.mdx
2. create leftCopy.mdx
3. do something like this:

```javascript
import LeftCopy from './leftCopy'
import BeforeCopy from './beforeCopy'

const CustomForm = () => {
  return <FormCK copyBefore={BeforeCopy} formId={...}>
    <BeforeCopy />
  </FormCK>
}
```

?

Reply:
You can go like this at the moment:

import LeftCopy from './leftCopy'
import BeforeCopy from './beforeCopy'

const CustomForm = () => {
  return <FormCK copyBefore={<BeforeCopy />} formId={...}>
    <LeftCopy />
  </FormCK>
}

Reply:
Perfect 😍